### ISSUE TITLE
Video Styling Enhancement

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1356]
**Component block:** video (left-align) and video (width-medium)

**Design:** https://www.figma.com/design/T2bgro0OiuLqxvm1uLhJZ2/Clark-County-NV?node-id=174-1683&t=zg8KECRBDzic3reI-0

As a site visitor, I want to be able to view youtube videos and understand the context of the video within the page so that I can learn more about a topic. 

**Requirements:** 
Enhance the video block with authorable eyebrow, title, and description fields
Text area has grey background color
Embed code for youtube video: _**metadata has to come from the embed code itself**_
Video plays inline (no change to current functionality, example: https://www.clarkcountynv.gov/government/departments/environment_and_sustainability/desert_conservation_program/video-library)
Video has proper spacing around the component block so that I can stack multiple videos on the page (design 1 for desktop) without the components touching
The left align video stacks to a single column on mobile with video top and text block below the video
The video block should be keyboard navigable
The video block should be screen reader accessible
If no text is added via authoring, default to today's display (ie no text column + no grey background)

**Acceptance Criteria:** 
1. Given I am adding a video block, when authoring the block, then I am able to add descriptive eyebrow, video title, and video description to display with the youtube embed. 
2. Given I am viewing a video block, when the page renders, then there is a grey background color in the text area of the block. 
3. Given I am viewing a video block, when I play the video, all settings and accessibility are included from the YouTube API so that I can easily view. 
4. Given I am viewing a video block, when I click play on the video, then it plays inline in the video area. 
5. Given I am authoring a page with videos, when I add multiple video blocks, then there is spacing between the blocks for visual distinctness. 
6. Given I am viewing a page with a video left-align, when I am on a mobile device, then the video block displays in a single column with video top. 
7. Given I am viewing a page with a video block, when I am using my keyboard to navigate, then I am able to navigate to, play/pause, and navigate the YouTube controls within the video to interact with the video.
8. Given I am viewing a video block, when there is a hyperlink included within the video block, then I am able to use my keyboard controls to navigate to and select the hyperlink within the video block text area. 
9. Given I am using a screen reader, when a page includes a video block, then the screen reader announces the video block content with the appropriate aria labels. 
10. Given I am using a screen reader, when a page includes a video block, then the screen reader announces the video block content in logical order.
11. Given I have not authored text into the block, when the component renders, then it displays only the video portion and no grey background column.

**Comment by sarah0arnold** [comment]
Add same element item styling for video text

**Comment by sarah0arnold** [comment]
Video reviewed in UAT session with project team 8/4/25 and passes with no feedback. Closing ticket
